### ISSUE TITLE
WP Admin menu duplicate

### ISSUE THREAD

**Issue by brown-a2** [ministryofjustice/intranet#237]
Create a new post for agency admins has "news" twice. One for news posts but one is for regional news posts. It needs to be relabeled and display permissions apply only to HMCTS.

**Comment by wilson1000-MoJ** [comment]
Hi @brown-a2, would you kindly confirm this is still an issue?
Many thanks in advance.

**Comment by brown-a2** [comment]
Yes, I've just had a look and can confirm this seems to have been resolved now. You can go ahead and close 👍
